libpod/container.go

```diff
@@ -471,6 +484,19 @@ func (c *Container) Start() error {
 		return err
 	}
 
+	mounted, err := mount.Mounted(c.config.ShmDir)
```

Comment:
I feel like we ought to make a generic Mount() function that ensures the container, SHM, and everything else are mounted and ready to go. But that can wait for another PR.